File: dojo_9/java/src/main/java/com/mercadolibre/dojos/Lugar.java
package com.mercadolibre.dojos;

public class Lugar {
    private final int km;

    public Lugar(String name, int km) {
        this.km = km;
    }

    public int distancia(Lugar destino) {
        return Math.abs(destino.km - this.km);
    }
}

File: dojo_kotlin/app/src/main/java/com/nelsito/kotlindojo9/Lugar.kt
package com.nelsito.kotlindojo9

/**
 * Created by npuglisi on 12/12/17.
 */
class Lugar(name: String, val km: Int) {
    fun distancia(destino: Lugar): Int {
        return Math.abs(destino.km - this.km)
    }
}